jdaviz/configs/specviz/plugins/unit_conversion/tests/test_unit_conversion.py

```diff
@@ -107,11 +73,12 @@ def test_no_spec_no_flux_uncert(specviz_app, spectrum1d):
 
 
 def test_no_spec_no_flux_uncert_unit_exp_none(specviz_app, spectrum1d):
+    np.random.seed(42)
     label = "Test 1D Spectrum"
     specviz_app.load_spectrum(spectrum1d, data_label=label)
 
     spectrum1d.uncertainty = UnknownUncertainty(np.abs(
-        np.random.randn(len(spectrum1d.spectral_axis.value))))
+        np.random.randn(len(spectrum1d.spectral_axis))))
```

Comment:
`astropy.units.Quantity` acts like Numpy array most of the time, so you can just call `len()` on it directly.